tests/admin_views/tests.py

```diff
@@ -67,6 +67,36 @@
 for a staff account. Note that both fields may be case-sensitive."
 
 
+class AdminFieldExtractionMixin(object):
+    """Mixin class for extracting data from iterables of `django.contrib.admin.helpers.AdminForm`."""
+
+    def get_admin_form_fields(self, admin_form):
+        """Extracting Fieldset, FieldLine and AdminField objects from AdminForm class."""
+        fieldsets = list(admin_form)
+
+        field_lines = []
+        for fs in fieldsets:
```

Comment:
Ok. I prefer abbreviated variable naming, when it does not worsen code readability, like this - https://github.com/django/django/blob/e519aab43a419589e92fe284e4ce2f2e034aec6a/django/core/management/commands/showmigrations.py#L100.